docs/src/robertson_benchmark.md

```diff
@@ -0,0 +1,272 @@
+
+```@example ROBER
+using Plots
+
+robertson_plot = function (sol, sol_ref = nothing, title = "")
+    colors = palette(:default)[1:3]'
+    if !isnothing(sol_ref)
+        p = plot(sol_ref, tspan = (1e-6, 1e11), xaxis = :log,
+                 idxs = [(0, 1), ((x, y) -> (x, 1e4 .* y), 0, 2), (0, 3)],
+                 linestyle = :dash, label = "", color = colors, linewidth = 2)
+        plot!(p, sol; tspan = (1e-6, 1e11), xaxis = :log, denseplot = false,
+              markers = :circle, ylims = (-0.2, 1.2),
+              idxs = [(0, 1), ((x, y) -> (x, 1e4 .* y), 0, 2), (0, 3)],
+              title, xticks = 10.0 .^ (-6:4:10), color = colors,
+              linewidht = 2, legend = :right, label = ["u₁" "u₂" "u₃"])
+    else
+        p = plot(sol; tspan = (1e-6, 1e11), xaxis = :log, denseplot = false,
+                 markers = :circle, ylims = (-0.2, 1.2),
+                 idxs = [(0, 1), ((x, y) -> (x, 1e4 .* y), 0, 2), (0, 3)],
+                 title, xticks = 10.0 .^ (-6:4:10), color = colors,
+                 linewidht = 2, legend = :right, label = ["u₁" "u₂" "u₃"])
+    end
+    return p
+end
+nothing # hide
+```
+
+For this stiff problem the computation of negative approximations may lead to inaccurate solutions. 
+This typically occurs when adaptive time stepping uses loose tolerances.
+
+```@example ROBER
+# compute reference solution for plotting
+ref_sol = solve(prob, Rodas4P(); abstol = 1e-14, reltol = 1e-13);
+
+# compute solutions with loose tolerances
+abstol = 1e-2
+reltol = 1e-1
+sol_Ros23 = solve(prob, Rosenbrock23(); abstol, reltol);
+sol_MPRK = solve(prob, MPRK22(1.0); abstol, reltol);
+
+# plot solutions
+p1 = robertson_plot(sol_Ros23, ref_sol, "Rosenbrock23");
+p2 = robertson_plot(sol_MPRK, ref_sol, "MPRK22(1.0)");
+plot(p1, p2)
+```
+
+Nevertheless, [OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/) provides the solver option `isoutofdomain`, which can be used in combination with [`isnegative`](@ref) to guarantee nonnegative solutions. 
+
+```@example ROBER
+sol_Ros23 = solve(prob, Rosenbrock23(); abstol, reltol, 
+                  isoutofdomain = isnegative) #reject negative solutions
+
+robertson_plot(sol_Ros23, ref_sol, "Rosenbrock23")
+```
+
+## Work-Precision diagrams
+
+In the following we show several work-precision diagrams, which compare different methods with respect to computing time and the respective error. 
+We focus solely on adaptive methods, since the time interval ``(0, 10^{11})`` is too large to generate accurate solutions with fixed step sizes.
+
+Since the Robertson problem is stiff, we need to use a suited implicit scheme to compute a reference solution, see the [solver guide](https://docs.sciml.ai/DiffEqDocs/dev/solvers/ode_solve/#Stiff-Problems). Note that we cannot use the recommended method `radau()`, since [`prob_pds_robertson`](@ref) uses [StaticArrays](https://juliaarrays.github.io/StaticArrays.jl/stable/) instead of arrays of type `Float64`.
```

Comment:
```suggestion
Since the Robertson problem is stiff, we need to use a suited implicit scheme to compute a reference solution, see the [solver guide](https://docs.sciml.ai/DiffEqDocs/dev/solvers/ode_solve/#Stiff-Problems).
Note that we cannot use the recommended method `radau()`, since [`prob_pds_robertson`](@ref) uses [StaticArrays.jl](https://juliaarrays.github.io/StaticArrays.jl/stable/) instead of standard `Array`s.
```